src/pages/about.jsx

```diff
@@ -36,90 +38,30 @@ export const About = () => {
-            />
-
-            <Card.Body>
-              <Card.Title>Port Townsend Hydrophone</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Bush Point Hydrophone</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Ocean Listening Exhibit</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Google summer of code</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Blog</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title> 2017 Orca Behavior Study</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Sailish Sea Vessel Research</Card.Title>
-            </Card.Body>
-          </Card>
-          <Card>
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Salish Sea Vessel Research</Card.Title>
-            </Card.Body>
-          </Card>
-        </CardColumns>
+        <Card sx={{ maxWidth: 345, marginBottom: '20px' }}>
+          <CardMedia
+            component="img"
+            height="140"
+            image="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
+          />
+          <CardContent>
+            <Typography gutterBottom variant="h5" component="div">
```

Comment:
```suggestion
            <Typography gutterBottom variant="h5">
```

Does it need to be a div? What's wrong with `h5`?